pybleau/app/plotting/plot_config.py

```diff
@@ -153,6 +153,9 @@ class BaseSinglePlotConfigurator(BasePlotConfigurator):
     #: Type of plot generated
     plot_type = Str
 
+    #: Template basis of plot (if exists)
+    template_basis = Either(None, Str, default=None)
```

Comment:
Changed the name to `source_template`.
The app should be able to check whether the `source_template` is `None`, instead of checking it is `Undefined` or an empty string or something else that is up for interpretation. If it is `None`, then there is no question that it hasn't been set.
That's my thought on it, anyway.

> What's the value of supporting `None` and not be able to assume it's a string no matter what? I would make this just a `Str`.

Reply:
Will change.